src/js/controller/controller.js

```diff
@@ -52,6 +52,7 @@ Object.assign(Controller.prototype, {
         let _actionOnAttach;
         let _stopPlaylist = false;
         let _interruptPlay;
+        let _stateOnScrubbing = null;
```

Comment:
@DanFerrer i don't think we need to store the state, since calling `_pause()` or `_play()` shouldn't alter the state; otherwise we'd have on pause events firing while scrubbing